src/systems/spawning/spawn_resource_planets.rs

```diff
@@ -28,11 +31,16 @@ pub fn spawn_resource_planets(
 
         transform.translation += transform.up() * PLANET_DISTANCE_FROM_SPACESTATION;
 
-        spawn_sprite_event_writer.send(SpawnSpriteEvent {
-            sprite_path: planet.sprite_path.to_string(),
-            size: planet.size,
-            transform,
-            entity: commands.spawn(planet).id(),
+        spawn_animated_sprite_event.send(SpawnAnimatedSpriteEvent {
```

Comment:
I know I should have made a new system in main then switched it in branch I am working it out still :scream: